### DISCUSSION TITLE
Can we have when clause context for snippet activation?

### DISCUSSION THREAD

**Discussion by adhamfarrag** [microsoft/vscode-discussions#1111] [Extension Development QnA] [answered]
[This discussion](https://github.com/microsoft/vscode-discussions/discussions/1024) should offer better context for my issue. But basically I want to be able to achieve partial feature activation using when clause context and part of that is snippets which doesn't support that. Can we have this feature? Where to ask for that ?

**Comment by isidorn** [comment, marked as answer]
Please file a new feature request here https://github.com/microsoft/vscode/issues
